quilt/test/build.yml

```diff
@@ -1,7 +1,11 @@
 ---
 contents:
   dataframes:
-    csv: [csv, data/10KRows13Cols.csv]
-    tsv: [tsv, data/10KRows13Cols.tsv]
-    xls: [xlsx, data/10KRows13Cols.xlsx]
-  README: [raw, data/README.md]
+    csv: 
+      file: data/10KRows13Cols.csv
```

Comment:
i looked at specifying a transform through auto-generate carefully and it's simpler not to touch that right now. i would need to pipe more function args to do it properly. i also think that a clean build.yml will give people a better first experience and they can learn about transform later.